test/org/sagebionetworks/bridge/services/StudyServiceMockTest.java

```diff
@@ -216,6 +234,115 @@ public void adminCannotSetActiveToFalse() {
     }
 
     @Test
+    public void createStudyAndUser() throws SynapseException {
```

Comment:
createStudy() and createSynapseProjectTeam() are already tested elswhere. It might be easiest to just spy StudyService, create mocks around createStudy() and createSynapse(), and just verify that these are called with the expected arguments.